internal/cloudapi/v2/server.go

```diff
@@ -130,17 +130,33 @@ func (s *Server) Handler(path string) http.Handler {
 		server: s,
 	}
 
+	// middlewares with auth
 	mws := []echo.MiddlewareFunc{
 		prometheus.StatusMiddleware(prometheus.ComposerSubsystem),
 	}
+	// middlewares without auth
+	mwsna := []echo.MiddlewareFunc{
+		prometheus.StatusMiddleware(prometheus.ComposerSubsystem),
+	}
 	if s.config.JWTEnabled {
 		mws = append(mws, auth.TenantChannelMiddleware(s.config.TenantProviderFields, HTTPError(ErrorTenantNotFound)))
 	}
-	mws = append(mws,
-		prometheus.HTTPDurationMiddleware(prometheus.ComposerSubsystem),
+	mws = append(mws, prometheus.HTTPDurationMiddleware(prometheus.ComposerSubsystem),
+		prometheus.MetricsMiddleware, s.ValidateRequest)
+	mwsna = append(mwsna, prometheus.HTTPDurationMiddleware(prometheus.ComposerSubsystem),
 		prometheus.MetricsMiddleware, s.ValidateRequest)
+
 	RegisterHandlers(e.Group(path, mws...), &handler)
 
+	// no auth endpoints
+	e.GET("/status", func(c echo.Context) error {
```

Comment:
I know right, but this is what is in the image-builder :-) Do you want me to break this pattern? I am fine with that, can also create both maybe?

Reply:
I think `/version` exists as it's in the `openapi.v2.yml` so this might already result in both existing (which is good)
Rather somehow assure `/version` not to be authenticated

Reply:
Ok added `/version` with non-auth middleware and also adding status/ready as fixed paths that simply return 200 without ANY middleware because we do not this to be logged or have metrics as k8s calls this every now and then.